keras_cv/models/segmentation/deeplab_v3_plus/deeplab_v3_plus_test.py

```diff
@@ -74,6 +74,18 @@ def test_weights_change(self):
             self.assertNotAllEqual(w1, w2)
             self.assertFalse(ops.any(ops.isnan(w2)))
 
+    @pytest.mark.large
+    def test_with_model_preset_forward_pass(self):
+        model = DeepLabV3Plus.from_preset(
+            "deeplab_v3_plus_resnet50_pascalvoc",
+            num_classes=21,
+            input_shape=[512, 512, 3],
+        )
+        image = np.ones((1, 512, 512, 3))
+        output = model(image)
+        expected_output = np.zeros((1, 512, 512, 3))
```

Comment:
Ahh I hadn't really considered that this would be the expected output, but of course this makes sense 😄 

Can you quickly verify that with randomly initialized weights this assertion would fail?

Reply:
That would still be all zeros - which is background. I tried to test with a little block in the image like this - 
```
image = np.zeros((512, 512,3), dtype=np.int8)
image[256:300, 256:300] = 230
```
None of this would match the 20+1 classes (personal, animal, items, etc) the model was trained on.

Reply:
Interesting -- maybe in the long run we should have a test image we use here, but for now this is fine I think.

Reply:
SG! Here is a test image I used for now to verify functionality
![image](https://github.com/keras-team/keras-cv/assets/78194266/45110cb0-0781-481c-8e81-5ec1b2b16f9d)